dash-pipeline/bmv2/dash_metadata.p4

```diff
@@ -38,9 +53,38 @@ struct eni_data_t {
     IPv4Address pl_underlay_sip;
 }
 
+typedef bit<32> RoutingType_t;
```

Comment:
please follow the naming convention and use `enum bit<32> dash_routing_type_t` instead.

Reply:
same as all other types, such as match stage, tunnel target and so on.